### PR TITLE
Added Serbian translation

### PR DESCRIPTION
**Pull request by dragstor**
Strings translated to Serbian, within a `values-sr` folder.

### CODE REVIEW

File: appintro/src/main/res/values-sr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <string name="app_intro_skip_button">PRESKOČI</string>
+    <string name="app_intro_next_button" tools:ignore="MissingTranslation">DALJE</string>
```
Comment (cortinico) [inline appintro/src/main/res/values-sr/strings.xml]: Can you remove the `tools:ignore="MissingTranslation"` from this line?

```suggestion
    <string name="app_intro_next_button">DALJE</string>
```

File: appintro/src/main/res/values-sr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    <string name="app_intro_back_button" tools:ignore="MissingTranslation">NAZAD</string>
```
Comment (cortinico) [inline appintro/src/main/res/values-sr/strings.xml]: Can you remove the `tools:ignore="MissingTranslation"` from this line?

```suggestion
    <string name="app_intro_back_button">NAZAD</string>
```

**Comment by dragstor** [comment]
Note: Strings are translated by using the Latin script, since it's used more than the Cyrillic one (in Apps).
 
How should we handle the Cyrillic script, since both are in use in Serbia?

- Latin script example: 
`Next -> DALJE`
- Cyrillic script example: 
`Next -> ДАЉЕ`

File: appintro/src/main/res/values-sr/strings.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    <string name="app_intro_done_button">KRAJ</string>
+    <string name="app_intro_image_content_description" tools:ignore="MissingTranslation">sličice</string>
```
Comment (cortinico) [inline appintro/src/main/res/values-sr/strings.xml]: Can you remove the `tools:ignore="MissingTranslation"` from this line?

```suggestion
    <string name="app_intro_image_content_description">sličice</string>
```

File: appintro/src/main/res/values-sr/strings.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
```
Comment (cortinico) [inline appintro/src/main/res/values-sr/strings.xml]: You should be fine also removing the `xmlns:tools` definition here.

**Comment by dragstor** [comment]
@cortinico cleanup done, waiting for Travis and then we're ready 👍 thanks!

**Comment by cortinico** [comment]
> How should we handle the Cyrillic script, since both are in use in Serbia?

IIRC Android doesn't have easy support for IETF BCP 47 (like `sr_Cyrl_RS` “Serbian (Cyrillic, Serbia)”). Ideally we should code something inside the library to handle this. I don't think it's probably worth the effort and I'd rather go with Latin only as you said: _since it's used more than the Cyrillic one (in Apps)._

**Comment by dragstor** [comment]
> > How should we handle the Cyrillic script, since both are in use in Serbia?
> 
> IIRC Android doesn't have easy support for IETF BCP 47 (like `sr_Cyrl_RS` “Serbian (Cyrillic, Serbia)”). Ideally we should code something inside the library to handle this. I don't think it's probably worth the effort and I'd rather go with Latin only as you said: _since it's used more than the Cyrillic one (in Apps)._

I agree. It was just me being curious :) 

Thanks for the merge btw 👍

**Comment by cortinico** [comment]
Thanks to you for the contrib :)

**Pull request merged**